### PR TITLE
Translation process of a project in Crowdin

### PR DESCRIPTION
**Pull request by Khadeeejah**
In my Pr i created `khadijah_docs.md` file in docs folder
i explained my translation process in Arabic language of a video subtitle

### CODE REVIEW

**Comment by Khadeeejah** [comment]
Hello @BatoolMM i have  submitted a pr . kindly review ma, thank you

**Comment by BatoolMM** [comment]
Thank you @Khadeeejah for your work, would you be able to correct the links in Markdown, please?

Screenshot 2022-10-12 at 19 31 43

**Comment by Khadeeejah** [comment]
I just fixed it @BatoolMM  thank you ma

**Comment by Khadeeejah** [comment]
Asalam Aleikum ma @BatoolMM  i am currently working on my final application and i would like to share my milestone/proposal for this particular project with you for your review to know if it goes in line with what the community wants to achieve at the end of the internship program.  Thanks in anticipation

**Comment by Khadeeejah** [comment]
Asalam Aleikum ma @BatoolMM i am currently working on my final application and i would like to share my milestone/proposal for this particular project with you for your review to know if it goes in line with what the community wants to achieve at the end of the internship program. Thanks in anticipation

**Comment by Khadeeejah** [comment]
Good morning @BatoolMM @alswajiab  the pull requests on this repo has not been accepted/ merged and we have to submit our contributions before tommorow. Should we just go ahead and submit our application ?

File: docs/khadijah_docs.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+
+##  تجربتي مع منصة الترجمة Crowdin
+
+تم إنشاء جمعية العلوم المفتوحة في المملكة العربية السعودية ** [OSCSA](https://osc-ksa.com/) تماشيًا مع الرؤية السعودية 2030 لدعم وتوفير بنية تحتية للعلوم المفتوحة في العالم العربي من خلال العمل مع الآخرين المجتمعات العلمية مثل Turing Way باستخدام [GitHub](www.github.com) ومنصة [Crowdin](www.crowdin.com)
+
+### البدء مع Crowdin
+
+<p> Crowdin هي أداة لإدارة التعريب تساعد الشركات والفرق والأفراد في الحفاظ على ذكائهم عند ترجمة منتجات مثل تطبيقات الأجهزة المحمولة وسطح المكتب والتطبيقات عبر الإنترنت والمواقع الإلكترونية والألعاب ومواد الدعم والمزيد.
```
Comment (BatoolMM) [inline docs/khadijah_docs.md]: This paragraph is aligned left to the right - it should be right to the left, can you fix it?

**Comment by BatoolMM** [comment]
@all-contrubutors please add @Khadeeejah for Translation and documentation

**Comment by BatoolMM** [comment]
@all-contributors please add @Khadeeejah for Translation and documentation

**Comment by BatoolMM** [comment]
As always, your work is fantastic, thank you so much!

**Pull request merged**